clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterRegistration.java

```diff
@@ -149,20 +152,29 @@ public void execute() {
     }
   }
 
-  private void dealIsSwaggerDifferent(Microservice newMicroservice) {
-    if (isListEquals(newMicroservice.getSchemas(), microservice.getSchemas())) {
-      return;
+  private boolean isSwaggerDifferent(Microservice newMicroservice) {
```

Comment:
Do not mix coverAndIgnore and isSwaggerDifferent, they have different meaning. 

Move it up

```
          boolean swaggerDifferent = isSwaggerDifferent(newMicroservice);
          if (swaggerDifferent  ){
             if( canRewriteSwagger()) {
             }...
          } else 
```